pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation_test.go

```diff
@@ -606,7 +584,10 @@ func TestExtractLibInfo(t *testing.T) {
 			},
 			containerRegistry: "registry",
 			expectedLibsToInject: []libInfo{
-				java.libInfo("", defaultLibImageVersions[java]),
```

Comment:
These changes were nice, but we can re-add them

Reply:
Agree